pgmpy/readwrite/XMLBIF.py

```diff
@@ -176,3 +177,202 @@ def get_model(self):
 
         model.add_cpds(*tabular_cpds)
         return model
+
+
+class XMLBIFWriter:
+    """
+    Base class for writing XMLBIF network file format.
+    """
+    def __init__(self, model, encoding='utf-8', prettyprint=True):
+        """
+        Initialise a XMLBIFWriter object.
+
+        Parameters
+        ----------
+        model: Model to write
+        encoding: String(optional)
+            Encoding for text data
+        prettyprint: Bool(optional)
+            Indentation in output XML if true
+
+        Examples
+        -------
+        >>> writer = XMLBIFWriter(model)
+        """
+        if not isinstance(model, BayesianModel):
+            raise TypeError("model must an instance of BayesianModel")
+        self.model = model
+
+        self.encoding = encoding
+        self.prettyprint = prettyprint
+
+        self.xml = etree.Element("BIF", attrib={'version': '0.3'})
+        self.network = etree.SubElement(self.xml, 'NETWORK')
+        try:
+            etree.SubElement(self.network, 'NAME').text = self.model['network_name']
+        except KeyError:
```

Comment:
The one that is there should work. The one I suggested may not work because an `nx.DiGraph` object apparently doesn't have a `get` method.